src/library_pthread.js

```diff
@@ -1350,6 +1350,11 @@ var LibraryPThread = {
     establishStackSpace(stackTop, stackMax);
   },
 
+  // allow pthreads to check if noExitRuntime from worker.js
+  $checkNoExitRuntime: function() {
```

Comment:
Maybe just `getNoExitRuntime` as this is a getter? "check" might imply assert or validate?

Reply:
Sounds good, renaming.